### ISSUE TITLE
IRAP Application

### ISSUE THREAD

**Issue by shervine** [USfoundation/org-design#9]
Need to meet with our ITA Rizwan to setup Us for IRAP support.

**Comment by shervine** [comment]
[IRAP_Funding_Opportunity_Overview_Template_-_2016.docx](https://github.com/USfoundation/org-design/files/912775/IRAP_Funding_Opportunity_Overview_Template_-_2016.docx)

**Comment by shervine** [comment]
@grumo I got this from Riz, our IRAP advisor. I am emailing Robert now...

-----------------------------------------

Hi Shervin,
 
Hope all is well and thanks for checking in with IRAP.
 
Yes I did review your [Y Combinator application](https://drive.google.com/file/d/0B4X48PHkwK8sTDliaXJ2dUFpNU0/view?usp=sharing) – very thin on any detail.  Based on that your firm is at a very early stage. However, there are a couple of comments.
 
My comments:
 
1. I don’t handle the EdTech space at IRAP – you would have to contact the NRC Concierge group (Robert McGarry - Robert.McGarry@nrc-cnrc.gc.ca ) and get a referral to the appropriate IRAP resource.
2. US-foundation looks to be a not for profit organization.  IRAP Tech Dev programs are for for-profit firms.
 
Cheers
 
Rizwan Kheraj, Ph.D.
Industrial Technology Advisor
Industrial  Research Assistance Program
National Research Council of Canada / Government of Canada    
E-mail: rizwan.kheraj@nrc-cnrc.gc.ca Tel: 604-323-5604     Fax: 604-666-7204

**Comment by shervine** [comment]
Hi Robert,

I got introduced to you via Rizwan Kheraj, our IRAP ITA.

We just started a non-profit technology Startup called [Us](https://drive.google.com/file/d/0B4X48PHkwK8sTDliaXJ2dUFpNU0/view), and we believe we can make a big impact on how intelligence is organized.

I would love to grab lunch or coffee, give you updates on our current stage and explore win/win opportunities.

​Cheers,​

Shervin Enayati
​​CEO​ & co-founder​
US Foundation
778.882.6962
